core/build.gradle.kts

```diff
@@ -386,3 +413,18 @@ tasks.withType<AbstractDokkaLeafTask>().configureEach {
         }
     }
 }
+
+// Disable intermediate sourceSet compilation because we do not need js-wasmJs artifact
+tasks.configureEach {
+    if (name == "compileJsAndWasmSharedMainKotlinMetadata") {
```

Comment:
While it disables the compilation, the information about the corresponding shared source set is still present in kotlin-project-structure-metadata.json embedded into common klib jar.
So it needs to be checked what effects it would have in IDE in projects having similar shared source sets.

Reply:
I used a such approach for all libraries and did't find any problems with it so far. The only problem that I found so far on the K1 IDE now reports error that it is no actual for JS and Wasm targets (in a same time it found these actuals in the shared sourceSet).